examples/README.md

```diff
@@ -54,12 +55,20 @@ For more details see the [Folder structure](#folder-structure) section.
 
 To preview the arbitary code example on CodeSandbox you must get the link to the project folder in the Gihub repo. You will find all code examples in our repo https://github.com/handsontable/handsontable/tree/develop/examples.
 
-For example: if the URL to the project is: https://github.com/handsontable/handsontable/tree/develop/examples/8.0.0/docs/js/settings you can preview that project on CodeSandbox adding the "box" word right after the "github" and before the ".com". The URL to the sandboxed project would be https://githubbox.com/handsontable/handsontable/tree/develop/examples/8.0.0/docs/js/settings.
+For example: if the URL to the project is: https://github.com/handsontable/handsontable/tree/code-examples-update/examples/next/docs/js/basic-example  you can preview that project on CodeSandbox adding the "box" word right after the "github" and before the ".com". The URL to the sandboxed project would be https://githubbox.com/handsontable/handsontable/tree/code-examples-update/examples/next/docs/js/basic-example .
```

Comment:
Linked to this branch which may be deleted. Should link to develop?

Reply:
Right 👍

Reply:
Done https://github.com/handsontable/handsontable/pull/7716/commits/c75cd1ddcac8002ee14c1ff8f57b9d4a0d31f581